### PR TITLE
Create a database based on a diagram

### PR DESCRIPTION
**Pull request by alvaroiporre**
##  Project requirements
In this PR we worked on the following:

- Given the following database diagram for a clinic:
clinic_diagram

- We created a file named schema_based_on_diagram.sql where we implemented the database from the diagram.
- Join tables from many-to-many relationships even they don't appear in the diagram.
- Added the FK indexes.

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  total_amount DECIMAL,
+  generated_at TIMESTAMP,
+  payed_at TIMESTAMP,
+  medical_history_id INTEGER UNIQUE REFERENCES medical_histories(id)
+);
+
+CREATE INDEX INDEX_FK_medical_history ON medical_history_treatments (medical_history_id);
+CREATE INDEX INDEX_FK_treatment_id ON medical_history_treatments (treatment_id);
+CREATE INDEX INDEX_FK_invoice_id ON invoice_items (invoice_id);
+CREATE INDEX INDEX_FK_treatment_id ON invoice_items (treatment_id);
```
Comment (Qoosim) [inline schema_based_on_diagram.sql]: - [ ] Great work for implementing your database from the given diagram. However, there're few things to add before you could move on. 
- [ ] The FK indexes implementation are yet to be complete. Kindly look into these two based on the given diagram.

```
  CREATE INDEX ON medical_histories (patient_id);
  CREATE INDEX ON invoices (medical_history_id);
```

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  Database created according to the given diagram. :heavy_check_mark: 
-  Correct Gitflow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (alvaroiporre) [reply schema_based_on_diagram.sql]: thank you for the feedback @Qoosim .

**Review by Ismailco — APPROVED** [review]
### Hi @alvaroiporre, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️ Used Gitflow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**